src/styles/nav.scss

```diff
@@ -27,6 +27,16 @@
   z-index: 100;
 }
 
+.side__social-links {
+  position: absolute;
+  right: 20px;
+  top: calc(350px + #{$navbar-height});
```

Comment:
can you also add a transition, so it's the same as the footer? can just copy-paste from the styling for the footer